miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -50,10 +50,11 @@ proc.authenticate_account_origin
 
     # assert that the caller is from the user context
     exec.account::authenticate_procedure
-    # => [CALLER, ...]
+    # => [storage_offset, ...]
 
-    # drop the caller
-    dropw
+    # TODO: use the storage_offset for storage access
+    # drop the storage_offset
+    drop
```

Comment:
Is the plan to implement this in a future PR?

Reply:
Yes, a second PR will be needed to close this issue: #667 using this offset to access storage.